Tests/eppo/ClientTests.swift

```diff
@@ -126,35 +103,40 @@ struct AssignmentTestCase : Decodable {
 }
 
 final class eppoClientTests: XCTestCase {
-   var loggerSpy: AssignmentLoggerSpy!
-   var eppoClient: EppoClient!
-   
-   override func setUpWithError() throws {
-       try super.setUpWithError()
+    var loggerSpy: AssignmentLoggerSpy!
+    var eppoClient: EppoClient!
+    
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+    
+       stub(condition: isHost("fscdn.eppo.cloud")) { _ in
+           let stubData = RacTestJSON.data(using: .utf8)!
+           return HTTPStubsResponse(data: stubData, statusCode: 200, headers: ["Content-Type": "application/json"])
+       }
+       
        loggerSpy = AssignmentLoggerSpy()
-       eppoClient = EppoClient("mock-api-key",
-                               host: "http://localhost:4001",
```

Comment:
safe to use the production host as its being intercepted

Reply:
Although if later somebody screws up mocking it will hit production 🤔

Reply:
That's true but it will fail with a bad API key - are you suggesting removing this library?